src/pages/home/HeaderView.js

```diff
@@ -93,6 +95,9 @@ const HeaderView = (props) => {
     const subtitle = getDefaultRoomSubtitle(props.report, props.policies);
     const isConcierge = participants.length === 1 && participants.includes(CONST.EMAIL.CONCIERGE);
 
+    // Except Concierge, call options for automated expensify accounts in 1:1 chat is not useful, so hide the button
+    const shouldShowCallButton = isConcierge || !(participants.length === 1 && hasExpensifyEmails(participants));
```

Comment:
NAB, I think this would be more clear as a variable so something like

```js
    const shouldShowCallButton = isConcierge || !isAutomatedExpensifyAccount; 
```

Reply:
>Except Concierge, call options for automated expensify accounts in 1:1 chat is not useful, so hide the button

NAB, the wording here is kind of weird I'd maybe change this to something like

>We hide the button when we are chatting with an automated Expensify account since it's not possible to contact these users via alternative means. It is possible to request a call with Concierge so we leave the option for them.

Reply:
Done